tests/local-integration-test.sh

```diff
@@ -33,7 +33,7 @@ clone:
 
 build:
   integration-test:
-    image: vmware-docker-ci-repo.bintray.io/integration/vic-test:1.1
+    image: vmware-docker-ci-repo.bintray.io/integration/vic-test:latest
```

Comment:
We don't want to do this. We want to specifically tie to a known version of the base image, otherwise you get weird results without any commit to the repo.